client/account/account.module.js

```diff
@@ -0,0 +1,35 @@
+const ng = require('angular');
+const accountCtrl = require('./account.ctrl');
+const passwordCtrl = require('./password.ctrl');
+const infoCtrl = require('./info.ctrl');
+const prefsCtrl = require('./prefs.ctrl');
+
+ng.module('porybox.account', ['ngRoute'])
+  .config(['$routeProvider', function($routeProvider) {
+    $routeProvider.when('/account', {
+      templateUrl: '/account/account.view.html',
+      controller: [accountCtrl],
+      controllerAs: 'account'
+    });
+  }])
+  .component('editPassword', {
+    templateUrl: 'account/password.view.html',
+    controller: ['io', '$mdToast', 'errorHandler', passwordCtrl],
+    controllerAs: 'password'
```

Comment:
Make sure to edit the `profile.something` statements to `password.something` in `client/account/password.view.html`. At the moment the form for changing your password doesn't work.